src/repositories/timesheet.rs

```diff
@@ -0,0 +1,126 @@
+use crate::common::DbResult;
+use chrono::{NaiveDate, NaiveDateTime};
+use sqlx::postgres::PgPool;
+use sqlx::prelude::FromRow;
+use std::sync::Arc;
+use uuid::Uuid;
+
+#[derive(Debug, FromRow)]
```

Comment:
Please, put these structs into a separate models.rs file.
Do it similarly to how the other repositories are defined/structured.